README.md

```diff
@@ -4,7 +4,12 @@ The main credential are automatically set up for you, howver, the **databaseURL*
 Use the following command to set the **databaseURL** & the **storageBucket** URLs:
   
 ```sh
-$ firebase functions:config:set audioseva.database_url="THE DATABASE URL" audioseva.storage_bucket="THE STORAGE BUCKET URL"
+$ firebase functions:config:set audioseva.database_url="THE DATABASE URL"
+$ firebase functions:config:set audioseva.storage_bucket="THE STORAGE BUCKET URL"
+$ firebase functions:config:set audioseva.send_in_blue.key="sendInBlue secret Key"
+# Coordinator details
+$ firebase functions:config:set audioseva.coordinator.templateid=NUMBER
+$ firebase functions:config:set audioseva.coordinator.email='EMAIL'
```

Comment:
```suggestion
$ firebase functions:config:set audioseva.coordinator.emailAddress='EMAIL'
```

Reply:
Config data can NOT have upper-cased names, can I replace it with something like this  **`coordinator.email_address`**